CONTRIBUTING.md

```diff
@@ -1,128 +1,102 @@
+documents, which has been confirmed through the IETF Last Call and IESG review
+stages. The goal of the Post-IESG process is to minimize the risk of changes
+invalidating the established consensus in these documents. Consequently, design
+changes will no longer be considered unless they relate to severe security,
+deployment or implementation problems. If you believe you have identified such a
+problem, please raise the issue on the [Working Group mailing
+list](https://www.ietf.org/mailman/listinfo/quic) with a clear marking in the
+`Subject:` line e.g. "New Issue ...".
 
-## Following Discussion
-
-The Working Group has a few venues for discussion:
-
-* We plan to meet at all [IETF meetings](https://www.ietf.org/meeting/) for the foreseeable future, and hold interim meetings between them. See our [meeting materials repository](https://github.com/quicwg/wg-materials) and the [official proceedings](https://datatracker.ietf.org/wg/quic/meetings/).
-
-* Our [mailing list](https://www.ietf.org/mailman/listinfo/quic) is used for most communication, including notifications of meetings, new drafts, consensus calls and other business, as well as issue discussion.
-
-* We also discuss specific issues on the appropriate issues list in [Github](https://github.com/quicwg/). If you don't want to use Github to follow these discussions, you can subscribe to the [issue announce list](https://www.ietf.org/mailman/listinfo/quic-issues).
-
-To be active in the Working Group, you can participate in any of these places. Most activity takes
-place on the mailing list, but if you just want to comment on and raise issues, that's fine too.
-
-
-## Raising Issues
+Any change will be subject to careful review and discussion, which might involve
+the editors, chairs, Working Group, and our Area Director.
 
-We use our [Github](https://github.com/quicwg/) issues lists to track items for discussion and
-their resolution.
-
-Before filing a new issue, please consider a few things:
-
-* Issues should be just that; issues with our deliverables, **not proposals, questions or support requests**.
-* Please review the issues list to make sure that you aren't filing a duplicate.
-* If you're not sure how to phrase your issue, please ask on the [mailing list](https://www.ietf.org/mailman/listinfo/quic).
-
-Issues can also be raised on the [Working Group mailing
-list](https://www.ietf.org/mailman/listinfo/quic) by clearly marking them as such (e.g., "New
-Issue" in the `Subject:` line).
-
-Be aware that issues might be rephrased, changed in scope, or combined with others, so that the
-group can focus its efforts. If you feel that such a change loses an important part of your
-original issue, please bring it up, either in comments or on the list.
-
-Off-topic and duplicate issues will be closed without discussion. Note that comments on individual
-commits will only be responded to with best effort, and may not be seen.
-
-
-## Resolving Issues
+### Resolving Issues
```

Comment:
I would keep cutting and just remove the issues and pull requests bit from here.